pkg/platform/aws/aws.go

```diff
@@ -287,6 +288,30 @@ func (c *config) checkValidConfig() hcl.Diagnostics {
 	return diagnostics
 }
 
+// checkDifferentPortForMuliplePools checks if user is using multiple worker
+// pools, then they should have lb_http_port and lb_https_port flag set
+// with different values.
+func (c *config) checkDifferentPortForMuliplePools() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	portMap := make(map[int]bool)
+
+	for _, wp := range c.WorkerPools {
+		if !portMap[wp.LBHTTPPort] && !portMap[wp.LBHTTPSPort] {
+			portMap[wp.LBHTTPPort] = true
+			portMap[wp.LBHTTPSPort] = true
+		} else {
+			diagnostics = append(diagnostics, &hcl.Diagnostic{
+				Severity: hcl.DiagError,
+				Summary:  "Unique values required",
+				Detail:   fmt.Sprintf("Values lb_http_port and lb_https_port not set for worker pool %s", wp.Name),
```

Comment:
IMO we should inform user which fields collide with itself to make debugging easier.